src/inspector/inspector_agent.cpp

```diff
@@ -42,9 +40,21 @@ std::string GetProcessTitle() {
 }
 
 std::string GenerateID() {
-  boost::uuids::uuid uuid = boost::uuids::random_generator()();
-  return boost::uuids::to_string(uuid);
-  //return "RANDON_UUID";
+  static std::random_device rd;
+  static std::mt19937 mte(rd());
+
+  std::uniform_int_distribution<int> dist(1, 60000);

```

Comment:
Why 60000? The default is max int if unspeified.

Reply:
Similarly, should we use uint16_t as a template parameter if that's what we're filling?